playbooks/roles/setup-caasp-workers/tasks/apparmor-profiles.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Copy openstack apparmor profile into the workers
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/apparmor.d/local/{{ item }}"
+  with_items: "{{ apparmor_profiles }}"
+  register: apparmor_copy
```

Comment:
Can you use

`notify:`

and handlers instead?

Reply:
nope, dynamic handlers do not work on ansible, nor does handlers with parameters, which kind of sucks

Reply:
What do you mean by dynamic handlers do not work on ansible?

Reply:
ok I did some hack around to manage those dynamic handlers, but IMO this is ugly AF.

Reply:
> What do you mean by dynamic handlers do not work on ansible?

something like:

```
- name: restart service {{ item }}
  service:
    name: {{ item }}
    state: restarted

```

so then you can just notify with `restart service WHATEVER` and get it done dynamically.

My hack kind of works but its really ugly, I think handlers should work more like this.

Reply:
Can be done with a chain of handlers and a set fact, but ok.

Reply:
That was a nit: anyway.